docker-compose.yml

```diff
@@ -37,7 +38,8 @@ services:
     command: ["bash", "/app/bin/startup.sh"]
 
   demo-express:
-    image: saasform/demo-express:latest
+    image: saasform/demo-express:localbuild
```

Comment:
Same as above

Reply:
I'm approving the PR and will do these update myself, to speed up